app/views/govuk_component/related_items.raw.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+  sections ||= []
+%>
+<aside class="govuk-related-items" role="complementary">
+  <% sections.each do |section| %>
+    <% section_title = section[:title] || "Elsewhere on GOV.UK" %>
```

Comment:
We discussed this and weren't sure what would be best so opted to maintain the status quo. The component would certainly be cleaner without it. What do you think?